server/services/interventionsService.ts

```diff
@@ -64,6 +64,16 @@ export interface CreateReportDateParams {
   toIncludingDate: CalendarDay
 }
 
+// Pagination parameters for resources that use spring pageable
```

Comment:
should this interface also live in the shared 'pagination' module?

Reply:
These are parameters just for the pagination request - all other params exist in this class too.

i.e.

```
export interface UpdateDraftEndOfServiceReportParams {
  furtherInformation: string | null
  outcome: CreateEndOfServiceReportOutcome | null
}
```